PianoFromAbove/Config.cpp

```diff
@@ -75,7 +75,7 @@ void Config::LoadConfigValues()
     LoadConfigValues( txRoot );
 
     // Custom settings need to be loaded from a separate file, otherwise stock PFA will reset them
-    doc = TiXmlDocument(sPath + "\\pfavizkhang.xml");
+    doc = TiXmlDocument(sPath + "\\Config-VIZ.xml");
```

Comment:
I don't see a reason to rename these. It'll force everyone to restore all of their configuration changes and it'll prevent settings from syncing with stock PFA.